pose_thumbnails/core.py

```diff
@@ -610,6 +618,41 @@ def _determine_poses(self):
             bpy.context.scene.tool_settings.use_keyframe_insert_auto = auto_insert
 
 
+class POSELIB_OT_sanitize_library_name(bpy.types.Operator):
+    """Make the active pose library name suitable for a character"""
```

Comment:
I think "sanitize" doesn't quite cover the semantics of this operator; "rename_for_character" covers it better IMO. You can then use that in the label as well, which gives you enough space in the docstring to explain how that's done (rather than explaining what "sanitize" means).